ncard_app/forms.py

```diff
@@ -16,38 +17,76 @@ def __init__(self, *args, **kwargs):
-            ),
-            Row('email', css_class='row'),
-            Row('email2', css_class='row'),
-            Row('phone_office', css_class='row'),
-            Row('phone_mobile', css_class='row'),
-            Row('phone_home', css_class='row'),
-            Row('cre_role', css_class='row'),
-            Row('ncard_relation', css_class='row'),
-            Row('project', css_class='row'),
-            Row('display_on_website', css_class='row'),
-            Row('profile_url', css_class='row'),
-            Row('orcid_id', css_class='row'),
-            Row('scopus_id', css_class='row'),
-            Row('wos_researcher_id', css_class='row'),
-            Row('google_scholar', css_class='row'),
-            Row('researchgate', css_class='row'),
-            Row('loop_profile', css_class='row'),
-            Row('linkedin', css_class='row'),
-            Row('twitter', css_class='row'),
-            Row('employers', css_class='row'),
-            Row('location', css_class='row'),
-            Row('organisation_primary', css_class='row'),
-            Row('organisation_other', css_class='row'),
-            Row('clinician', css_class='row'),
-            Row('notes', css_class='row'),
-            Row('research_focus', css_class='row'),
+            TabHolder(
+                Tab('Contact Details',
+                    HTML("<br>"),
+                    Row(
+                        Column('title', css_class='col-md-1 mb-0'),
+                        Column('given_name', css_class='col-md-4 mb-0'),
+                        Column('middle_name', css_class='col-md-3 mb-0'),
+                        Column('surname', css_class='col-md-4 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('email', css_class='row'),
+                        Column('email2', '', css_class='row'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('phone_office', css_class='row'),
+                        Column('phone_mobile', css_class='row'),
+                        Column('phone_home', css_class='row'),
+                        css_class='row'
+                    ),
+                ),
+                
+                Tab('NCARD Info',
+                    HTML("<br>"),
+                    Row(
+                        Column('cre_role', css_class='col-md-4 mb-0'),
+                        Column('ncard_relation', css_class='col-md-4 mb-0'),
+                        Column('project', css_class='col-md-4 mb-0'),
+                        css_class='row'
+                    ),
+                    Row(
+                        Column('display_on_website', css_class='col-md-3 mb-0'),
+                        Column('profile_url', css_class='col-md-9 mb-0'),
+                        css_class='row'
+                    ),
+                ),
+
+                Tab('Individal Reseracher Details',
```

Comment:
This was a typo, and also not a good title in my opinion.

```suggestion
                Tab('Researcher Profile',
```

Reply:
There is still a typo in "Reseracher".